packages/variants/README.md

```diff
@@ -1,11 +1,15 @@
 # [variants](http://dev.office.com/fabric)
 
-Variants are themes based on another theme, subthemes which share the same colors as the original theme, but with those colors used in different places. A variant is still a full theme in and of itself.
+Variants are [themes](https://github.com/OfficeDev/office-ui-fabric-react/blob/master/ghdocs/Theming.md) generated from an existing theme (as opposed to [from raw colors](https://developer.microsoft.com/en-us/fabric#/styles/themegenerator)). A variant will share the same colors as the original theme it was generated from, but will use those colors differently. For example, the background color and text color might be swapped. Variants can be used to highlight or de-emphasize portions of the page.
```

Comment:
You don't need parens, just add a comma so something like `existing theme, as opposed to ...`

Reply:
but i (REALLY) like parens!!